dev/TEAL.md

```diff
@@ -732,25 +740,25 @@ A compiled program starts with a varuint declaring the version of the compiled c
 
 For version 1, subsequent bytes after the varuint are program opcode bytes. Future versions could put other metadata following the version identifier.
 
-It is important to prevent newly-introduced transaction fields from
-breaking assumptions made by older versions of the AVM. If one of the
-transactions in a group will execute a program whose version predates
-a given field, that field must not be set anywhere in the transaction
-group, or the group will be rejected. For example, executing a version
-1 program on a transaction with RekeyTo set to a nonzero address will
-cause the program to fail, regardless of the other contents of the
-program itself.
+It is important to prevent newly-introduced transaction types and
+fields from breaking assumptions made by programs written before they
+existed. If one of the transactions in a group will execute a program
+whose version predates a transaction type or field that can violate
+expectations, that transaction type or field must not be used anywhere
+in the transaction group. A v1 program included in a transaction group
```

Comment:
```suggestion
in the transaction group. 

Concretely, the above requirement is translated as follows:
A v1 program included in a transaction group
```

Reply:
I've committed that. If this seems good, please approve but hold off on merging, as per @algorandskiy

Reply:
I withdraw my prev request to hold the merge - I've used the block size increase commit hash for v33 and any other final merge can be used for v34. Please go ahead and merge if it looks good.